artview/parser.py

```diff
@@ -7,10 +7,10 @@
 import argparse
 
 # Get the version
-f = open("version", "r")
-NAME = f.readline().strip()
-VERSION = f.readline().strip()
-f.close()
+import version
```

Comment:
this will be created while installing, for instance in place with `python setup.py build_ext -i`. We need to add some install instructions